roles/zookeeper-docker/defaults/main.yml

```diff
@@ -1,6 +1,33 @@
 zookeeper_client_port: 2181
 zookeeper_leader_connect_port: 2888
 zookeeper_leader_election_port: 3888
+zookeeper_init_limit: 5
+zookeeper_sync_limit: 2
+zookeeper_tick_time: 2000
+zookeeper_data_dir: /data
+zookeeper_log_dir: /datalog
+
+# specifies what will container use for data/log directories
+# possible values are:
+#   "container" - use directories on docker container instance (default)
```

Comment:
And if we leave out the prefix => containers, dirs, implicit. :) Not that different than what we already have. Just add better descriptions.